argo-cd-apps/overlays/development/delete-applications.yaml

```diff
@@ -34,3 +34,10 @@ kind: ApplicationSet
 metadata:
   name: cluster-secret-store
 $patch: delete
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: segment-bridge
+$patch: delete
```

Comment:
Does this mean that it will not be deployed in dev? If so, do we need to maintain the dev configs under the `segment-bridge` component?

Reply:
While this is a mistake I'll fix, The current requirement is for the staging cluster, so if we'll start not including this in envs, prod should be selected as well.

Reply:
ok. Do you want it on all stage clusters? I thought you mentioned you want it on a specific cluster.

Reply:
Staging host cluster is what we need. is differentiation possible in those files?

Reply:
Yes. There are ways to differentiate between the different clusters, but as far as I understand, the starting point is the collection of clusters it will be deployed on. I think currently, you're deploying only to member clusters.

Have a look here: https://github.com/redhat-appstudio/infra-deployments/tree/main/argo-cd-apps/base

Your configs are under `member`. I think it means that it will only be deployed to member clusters. You can either have it under `all-clusters` and then differentiate by name like we did for the prometheus configs, or you can have it under `host` and then I think it will only go to the host cluster.